src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/PairedEndAndSplitReadEvidenceCollection.java

```diff
@@ -176,19 +229,23 @@ private void flushDiscordantReadPairs() {
     }
 
     private void writeDiscordantPair(final DiscordantRead r) {
-        final boolean strandA = !r.isReadReverseStrand();
-        final boolean strandB = !r.isMateReverseStrand();
+        final String strandA = r.isReadReverseStrand() ? "-" : "+";
+        final String strandB = r.isMateReverseStrand() ? "-" : "+";
 
-        final DiscordantPairEvidence discordantPair = new DiscordantPairEvidence(sampleName, r.getContig(), r.getStart(), strandA, r.getMateContig(), r.getMateStart(), strandB);
-        peWriter.add(discordantPair);
+        try {
+            // subtract 1 from positions to match pysam output
+            peWriter.write(r.getContig() + "\t" + (r.getStart() - 1) + "\t" + strandA + "\t" + r.getMateContig() + "\t" + (r.getMateStart() - 1) + "\t" + strandB + "\t" + sampleName + "\n");
+        } catch (IOException e) {
+            throw new GATKException("Could not write to PE file", e);
+        }
```

Comment:
This looks like recently changed code from commit `b68eb8778d65f84c46f2e1fe44683e861bfd67d2` on master (and elsewhere in this file). Can you check for accidental reversions?